docs/start/considerations.md

```diff
@@ -0,0 +1,54 @@
+---
+id: considerations
+title: Practical Considerations
+---
+
+Having understood the requirements for running a full node or a smeshing node, it is important to take note of the following considerations as you embark on your smeshing journey.
+
+### CPU Usage
+
+As long as your system and configuration meet the recommended requirements, the on-going utilization of your CPU by the Smapp should be limited to 2 CPU cores during normal operation (after the initial setup phase).
+
+### Network Configuration
+
+Your network should allow the Smapp to accept incoming connections on **UDP port 7153** and on **TCP port 7153** and your computer firewall **should not block incoming UDP and TPC packets on this port**. The App attempts to automatically configure your network using UPnP. In some cases you may need to configure your router and firewall manually. Follow [this guide](./smeshing/netconfig.md) to configure your network.
+
+### Using a Laptop for Smeshing
+
+Of course, you can smesh on your laptop if it meets the minimum system requirements. However, you will need to make sure you leave your laptop **open/awake and connected to power and to the Internet 24/7**. This may degrade your laptop's battery.
+
+Occasionally **shutting down your laptop or the Smapp may prevent you from Smeshing blocks and from earning Smeshing rewards**.
+
+### PoS Data
+
+You only need to setup proof of space data once to smesh for an unlimited period of time. The larger your proof of space data is, the higher your smeshing rewards will be.
+
+You can use any magnetic hard disk drive (HDD) or solid state drive (SSD) for storing the PoS data. There is no significant advantage in using an SSD over HDD. 
+
+You do not need to have temporary disk space larger than your final proof of space data. Creating proof of space data on more than one volume is not yet supported.
+
+### Network Operations
+
+The biggest resource consumed by a Spacemesh node is bandwidth. A full node currently consumes between 150-1500mb/hour of bandwidth. The node is quiescent most of the time, consuming only 100-300kbit/s, but spikes as high as 35Mbit/s every few minutes when a new layer appears and network traffic increases (Hare messages, proposals, etc.). Bandwidth is variable and depends heavily on the network condition and the number and quality of one's peers.
```

Comment:
I think given that compact encoding is on the way and hare committee reduction, too, we can skip that here.